kong/runloop/wasm.lua

```diff
@@ -48,6 +72,8 @@ local TYPE_COMBINED = 2
 
 
 local ENABLED = false
+local STATUS = "wasm support is not enabled"
```

Comment:
nit: "wasm support is not enabled" seems has been repeated several times in this file. I'm thinking whether we will have const variable to store this string. But it seems `const` is only available in Lua 5.4.

Reply:
while we're at it, I wasn't going to comment on style preferences, but... (—the best way to start a review comment, right? :laughing: ) at first glance the all-uppercase variables do look like they're intended as constants. If they're meant as internal status-keeping variables, perhaps something like `local wasm_enabled` and `local wasm_status` might be easier to understand at a glance (and then you could do the likes of `local wasm_status = WASM_STATUS_NOT_ENABLED` if you wanted to centralize the messages at the top). Just a thought!

Reply:
That's fair. I've moved the status strings into local, "constant"-y `STATUS_*` vars at the module level.

Reply:
Ugh, yeah I wish Lua were better about this. I like my `SCREAMING_SNAKE_CASE` for constants, but I _also_ like it for "this variable is from the outermost scope in the file"